tests/e2e.test.ts

```diff
@@ -52,18 +52,7 @@ describe('E2E tests', function () {
     }
     toolCommand = 'gcloud';
     if (SERVICE) {
-      sleep(10000);
```

Comment:
I removed this since it wasn't actually doing anything. As @verbanicm pointed out, it was missing an `await` and therefore was never actually needed.